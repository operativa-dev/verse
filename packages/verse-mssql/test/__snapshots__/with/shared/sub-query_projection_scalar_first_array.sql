-- Executing SQL: Parameters: []
select (
   select "t4"."Title"
   from "Album" as "t4"
   order by 1 offset 0 rows fetch next 1 rows only
) as "c0", "t1"."Name"
from "Artist" as "t1"

