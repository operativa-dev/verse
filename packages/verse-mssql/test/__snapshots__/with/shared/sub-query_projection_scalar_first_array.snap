[
  [
    "...And Justice For All",
    "AC/DC",
  ],
  [
    "...And Justice For All",
    "Accept",
  ],
  [
    "...And Justice For All",
    "Aerosmith",
  ],
  [
    "...And Justice For All",
    "Alanis Morissette",
  ],
  [
    "...And Justice For All",
    "Alice In Chains",
  ],
  [
    "...And Justice For All",
    "Antônio Carlos Jobim",
  ],
  [
    "...And Justice For All",
    "Apocalyptica",
  ],
  [
    "...And Justice For All",
    "Audioslave",
  ],
  [
    "...And Justice For All",
    "BackBeat",
  ],
  [
    "...And Justice For All",
    "Billy Cobham",
  ],
  [
    "...And Justice For All",
    "Black Label Society",
  ],
  [
    "...And Justice For All",
    "Black Sabbath",
  ],
  [
    "...And Justice For All",
    "Body Count",
  ],
  [
    "...And Justice For All",
    "Bruce Dickinson",
  ],
  [
    "...And Justice For All",
    "Buddy Guy",
  ],
  [
    "...And Justice For All",
    "Caetano Veloso",
  ],
  [
    "...And Justice For All",
    "Chico Buarque",
  ],
  [
    "...And Justice For All",
    "Chico Science & Nação Zumbi",
  ],
  [
    "...And Justice For All",
    "Cidade Negra",
  ],
  [
    "...And Justice For All",
    "Cláudio Zoli",
  ],
  [
    "...And Justice For All",
    "Various Artists",
  ],
  [
    "...And Justice For All",
    "Led Zeppelin",
  ],
  [
    "...And Justice For All",
    "Frank Zappa & Captain Beefheart",
  ],
  [
    "...And Justice For All",
    "Marcos Valle",
  ],
  [
    "...And Justice For All",
    "Milton Nascimento & Bebeto",
  ],
  [
    "...And Justice For All",
    "Azymuth",
  ],
  [
    "...And Justice For All",
    "Gilberto Gil",
  ],
  [
    "...And Justice For All",
    "João Gilberto",
  ],
  [
    "...And Justice For All",
    "Bebel Gilberto",
  ],
  [
    "...And Justice For All",
    "Jorge Vercilo",
  ],
  [
    "...And Justice For All",
    "Baby Consuelo",
  ],
  [
    "...And Justice For All",
    "Ney Matogrosso",
  ],
  [
    "...And Justice For All",
    "Luiz Melodia",
  ],
  [
    "...And Justice For All",
    "Nando Reis",
  ],
  [
    "...And Justice For All",
    "Pedro Luís & A Parede",
  ],
  [
    "...And Justice For All",
    "O Rappa",
  ],
  [
    "...And Justice For All",
    "Ed Motta",
  ],
  [
    "...And Justice For All",
    "Banda Black Rio",
  ],
  [
    "...And Justice For All",
    "Fernanda Porto",
  ],
  [
    "...And Justice For All",
    "Os Cariocas",
  ],
  [
    "...And Justice For All",
    "Elis Regina",
  ],
  [
    "...And Justice For All",
    "Milton Nascimento",
  ],
  [
    "...And Justice For All",
    "A Cor Do Som",
  ],
  [
    "...And Justice For All",
    "Kid Abelha",
  ],
  [
    "...And Justice For All",
    "Sandra De Sá",
  ],
  [
    "...And Justice For All",
    "Jorge Ben",
  ],
  [
    "...And Justice For All",
    "Hermeto Pascoal",
  ],
  [
    "...And Justice For All",
    "Barão Vermelho",
  ],
  [
    "...And Justice For All",
    "Edson, DJ Marky & DJ Patife Featuring Fernanda Porto",
  ],
  [
    "...And Justice For All",
    "Metallica",
  ],
  [
    "...And Justice For All",
    "Queen",
  ],
  [
    "...And Justice For All",
    "Kiss",
  ],
  [
    "...And Justice For All",
    "Spyro Gyra",
  ],
  [
    "...And Justice For All",
    "Green Day",
  ],
  [
    "...And Justice For All",
    "David Coverdale",
  ],
  [
    "...And Justice For All",
    "Gonzaguinha",
  ],
  [
    "...And Justice For All",
    "Os Mutantes",
  ],
  [
    "...And Justice For All",
    "Deep Purple",
  ],
  [
    "...And Justice For All",
    "Santana",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Dave Matthews",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Everlast",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Rob Thomas",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Lauryn Hill & Cee-Lo",
  ],
  [
    "...And Justice For All",
    "Santana Feat. The Project G&B",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Maná",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Eagle-Eye Cherry",
  ],
  [
    "...And Justice For All",
    "Santana Feat. Eric Clapton",
  ],
  [
    "...And Justice For All",
    "Miles Davis",
  ],
  [
    "...And Justice For All",
    "Gene Krupa",
  ],
  [
    "...And Justice For All",
    "Toquinho & Vinícius",
  ],
  [
    "...And Justice For All",
    "Vinícius De Moraes & Baden Powell",
  ],
  [
    "...And Justice For All",
    "Vinícius De Moraes",
  ],
  [
    "...And Justice For All",
    "Vinícius E Qurteto Em Cy",
  ],
  [
    "...And Justice For All",
    "Vinícius E Odette Lara",
  ],
  [
    "...And Justice For All",
    "Vinicius, Toquinho & Quarteto Em Cy",
  ],
  [
    "...And Justice For All",
    "Creedence Clearwater Revival",
  ],
  [
    "...And Justice For All",
    "Cássia Eller",
  ],
  [
    "...And Justice For All",
    "Def Leppard",
  ],
  [
    "...And Justice For All",
    "Dennis Chambers",
  ],
  [
    "...And Justice For All",
    "Djavan",
  ],
  [
    "...And Justice For All",
    "Eric Clapton",
  ],
  [
    "...And Justice For All",
    "Faith No More",
  ],
  [
    "...And Justice For All",
    "Falamansa",
  ],
  [
    "...And Justice For All",
    "Foo Fighters",
  ],
  [
    "...And Justice For All",
    "Frank Sinatra",
  ],
  [
    "...And Justice For All",
    "Funk Como Le Gusta",
  ],
  [
    "...And Justice For All",
    "Godsmack",
  ],
  [
    "...And Justice For All",
    "Guns N' Roses",
  ],
  [
    "...And Justice For All",
    "Incognito",
  ],
  [
    "...And Justice For All",
    "Iron Maiden",
  ],
  [
    "...And Justice For All",
    "James Brown",
  ],
  [
    "...And Justice For All",
    "Jamiroquai",
  ],
  [
    "...And Justice For All",
    "JET",
  ],
  [
    "...And Justice For All",
    "Jimi Hendrix",
  ],
  [
    "...And Justice For All",
    "Joe Satriani",
  ],
  [
    "...And Justice For All",
    "Jota Quest",
  ],
  [
    "...And Justice For All",
    "João Suplicy",
  ],
  [
    "...And Justice For All",
    "Judas Priest",
  ],
  [
    "...And Justice For All",
    "Legião Urbana",
  ],
  [
    "...And Justice For All",
    "Lenny Kravitz",
  ],
  [
    "...And Justice For All",
    "Lulu Santos",
  ],
  [
    "...And Justice For All",
    "Marillion",
  ],
  [
    "...And Justice For All",
    "Marisa Monte",
  ],
  [
    "...And Justice For All",
    "Marvin Gaye",
  ],
  [
    "...And Justice For All",
    "Men At Work",
  ],
  [
    "...And Justice For All",
    "Motörhead",
  ],
  [
    "...And Justice For All",
    "Motörhead & Girlschool",
  ],
  [
    "...And Justice For All",
    "Mônica Marianno",
  ],
  [
    "...And Justice For All",
    "Mötley Crüe",
  ],
  [
    "...And Justice For All",
    "Nirvana",
  ],
  [
    "...And Justice For All",
    "O Terço",
  ],
  [
    "...And Justice For All",
    "Olodum",
  ],
  [
    "...And Justice For All",
    "Os Paralamas Do Sucesso",
  ],
  [
    "...And Justice For All",
    "Ozzy Osbourne",
  ],
  [
    "...And Justice For All",
    "Page & Plant",
  ],
  [
    "...And Justice For All",
    "Passengers",
  ],
  [
    "...And Justice For All",
    "Paul D'Ianno",
  ],
  [
    "...And Justice For All",
    "Pearl Jam",
  ],
  [
    "...And Justice For All",
    "Peter Tosh",
  ],
  [
    "...And Justice For All",
    "Pink Floyd",
  ],
  [
    "...And Justice For All",
    "Planet Hemp",
  ],
  [
    "...And Justice For All",
    "R.E.M. Feat. Kate Pearson",
  ],
  [
    "...And Justice For All",
    "R.E.M. Feat. KRS-One",
  ],
  [
    "...And Justice For All",
    "R.E.M.",
  ],
  [
    "...And Justice For All",
    "Raimundos",
  ],
  [
    "...And Justice For All",
    "Raul Seixas",
  ],
  [
    "...And Justice For All",
    "Red Hot Chili Peppers",
  ],
  [
    "...And Justice For All",
    "Rush",
  ],
  [
    "...And Justice For All",
    "Simply Red",
  ],
  [
    "...And Justice For All",
    "Skank",
  ],
  [
    "...And Justice For All",
    "Smashing Pumpkins",
  ],
  [
    "...And Justice For All",
    "Soundgarden",
  ],
  [
    "...And Justice For All",
    "Stevie Ray Vaughan & Double Trouble",
  ],
  [
    "...And Justice For All",
    "Stone Temple Pilots",
  ],
  [
    "...And Justice For All",
    "System Of A Down",
  ],
  [
    "...And Justice For All",
    "Terry Bozzio, Tony Levin & Steve Stevens",
  ],
  [
    "...And Justice For All",
    "The Black Crowes",
  ],
  [
    "...And Justice For All",
    "The Clash",
  ],
  [
    "...And Justice For All",
    "The Cult",
  ],
  [
    "...And Justice For All",
    "The Doors",
  ],
  [
    "...And Justice For All",
    "The Police",
  ],
  [
    "...And Justice For All",
    "The Rolling Stones",
  ],
  [
    "...And Justice For All",
    "The Tea Party",
  ],
  [
    "...And Justice For All",
    "The Who",
  ],
  [
    "...And Justice For All",
    "Tim Maia",
  ],
  [
    "...And Justice For All",
    "Titãs",
  ],
  [
    "...And Justice For All",
    "Battlestar Galactica",
  ],
  [
    "...And Justice For All",
    "Heroes",
  ],
  [
    "...And Justice For All",
    "Lost",
  ],
  [
    "...And Justice For All",
    "U2",
  ],
  [
    "...And Justice For All",
    "UB40",
  ],
  [
    "...And Justice For All",
    "Van Halen",
  ],
  [
    "...And Justice For All",
    "Velvet Revolver",
  ],
  [
    "...And Justice For All",
    "Whitesnake",
  ],
  [
    "...And Justice For All",
    "Zeca Pagodinho",
  ],
  [
    "...And Justice For All",
    "The Office",
  ],
  [
    "...And Justice For All",
    "Dread Zeppelin",
  ],
  [
    "...And Justice For All",
    "Battlestar Galactica (Classic)",
  ],
  [
    "...And Justice For All",
    "Aquaman",
  ],
  [
    "...And Justice For All",
    "Christina Aguilera featuring BigElf",
  ],
  [
    "...And Justice For All",
    "Aerosmith & Sierra Leone's Refugee Allstars",
  ],
  [
    "...And Justice For All",
    "Los Lonely Boys",
  ],
  [
    "...And Justice For All",
    "Corinne Bailey Rae",
  ],
  [
    "...And Justice For All",
    "Dhani Harrison & Jakob Dylan",
  ],
  [
    "...And Justice For All",
    "Jackson Browne",
  ],
  [
    "...And Justice For All",
    "Avril Lavigne",
  ],
  [
    "...And Justice For All",
    "Big & Rich",
  ],
  [
    "...And Justice For All",
    "Youssou N'Dour",
  ],
  [
    "...And Justice For All",
    "Black Eyed Peas",
  ],
  [
    "...And Justice For All",
    "Jack Johnson",
  ],
  [
    "...And Justice For All",
    "Ben Harper",
  ],
  [
    "...And Justice For All",
    "Snow Patrol",
  ],
  [
    "...And Justice For All",
    "Matisyahu",
  ],
  [
    "...And Justice For All",
    "The Postal Service",
  ],
  [
    "...And Justice For All",
    "Jaguares",
  ],
  [
    "...And Justice For All",
    "The Flaming Lips",
  ],
  [
    "...And Justice For All",
    "Jack's Mannequin & Mick Fleetwood",
  ],
  [
    "...And Justice For All",
    "Regina Spektor",
  ],
  [
    "...And Justice For All",
    "Scorpions",
  ],
  [
    "...And Justice For All",
    "House Of Pain",
  ],
  [
    "...And Justice For All",
    "Xis",
  ],
  [
    "...And Justice For All",
    "Nega Gizza",
  ],
  [
    "...And Justice For All",
    "Gustavo & Andres Veiga & Salazar",
  ],
  [
    "...And Justice For All",
    "Rodox",
  ],
  [
    "...And Justice For All",
    "Charlie Brown Jr.",
  ],
  [
    "...And Justice For All",
    "Pedro Luís E A Parede",
  ],
  [
    "...And Justice For All",
    "Los Hermanos",
  ],
  [
    "...And Justice For All",
    "Mundo Livre S/A",
  ],
  [
    "...And Justice For All",
    "Otto",
  ],
  [
    "...And Justice For All",
    "Instituto",
  ],
  [
    "...And Justice For All",
    "Nação Zumbi",
  ],
  [
    "...And Justice For All",
    "DJ Dolores & Orchestra Santa Massa",
  ],
  [
    "...And Justice For All",
    "Seu Jorge",
  ],
  [
    "...And Justice For All",
    "Sabotage E Instituto",
  ],
  [
    "...And Justice For All",
    "Stereo Maracana",
  ],
  [
    "...And Justice For All",
    "Cake",
  ],
  [
    "...And Justice For All",
    "Aisha Duo",
  ],
  [
    "...And Justice For All",
    "Habib Koité and Bamada",
  ],
  [
    "...And Justice For All",
    "Karsh Kale",
  ],
  [
    "...And Justice For All",
    "The Posies",
  ],
  [
    "...And Justice For All",
    "Luciana Souza/Romero Lubambo",
  ],
  [
    "...And Justice For All",
    "Aaron Goldberg",
  ],
  [
    "...And Justice For All",
    "Nicolaus Esterhazy Sinfonia",
  ],
  [
    "...And Justice For All",
    "Temple of the Dog",
  ],
  [
    "...And Justice For All",
    "Chris Cornell",
  ],
  [
    "...And Justice For All",
    "Alberto Turco & Nova Schola Gregoriana",
  ],
  [
    "...And Justice For All",
    "Richard Marlow & The Choir of Trinity College, Cambridge",
  ],
  [
    "...And Justice For All",
    "English Concert & Trevor Pinnock",
  ],
  [
    "...And Justice For All",
    "Anne-Sophie Mutter, Herbert Von Karajan & Wiener Philharmoniker",
  ],
  [
    "...And Justice For All",
    "Hilary Hahn, Jeffrey Kahane, Los Angeles Chamber Orchestra & Margaret Batjer",
  ],
  [
    "...And Justice For All",
    "Wilhelm Kempff",
  ],
  [
    "...And Justice For All",
    "Yo-Yo Ma",
  ],
  [
    "...And Justice For All",
    "Scholars Baroque Ensemble",
  ],
  [
    "...And Justice For All",
    "Academy of St. Martin in the Fields & Sir Neville Marriner",
  ],
  [
    "...And Justice For All",
    "Academy of St. Martin in the Fields Chamber Ensemble & Sir Neville Marriner",
  ],
  [
    "...And Justice For All",
    "Berliner Philharmoniker, Claudio Abbado & Sabine Meyer",
  ],
  [
    "...And Justice For All",
    "Royal Philharmonic Orchestra & Sir Thomas Beecham",
  ],
  [
    "...And Justice For All",
    "Orchestre Révolutionnaire et Romantique & John Eliot Gardiner",
  ],
  [
    "...And Justice For All",
    "Britten Sinfonia, Ivor Bolton & Lesley Garrett",
  ],
  [
    "...And Justice For All",
    "Chicago Symphony Chorus, Chicago Symphony Orchestra & Sir Georg Solti",
  ],
  [
    "...And Justice For All",
    "Sir Georg Solti & Wiener Philharmoniker",
  ],
  [
    "...And Justice For All",
    "Academy of St. Martin in the Fields, John Birch, Sir Neville Marriner & Sylvia McNair",
  ],
  [
    "...And Justice For All",
    "London Symphony Orchestra & Sir Charles Mackerras",
  ],
  [
    "...And Justice For All",
    "Barry Wordsworth & BBC Concert Orchestra",
  ],
  [
    "...And Justice For All",
    "Herbert Von Karajan, Mirella Freni & Wiener Philharmoniker",
  ],
  [
    "...And Justice For All",
    "Eugene Ormandy",
  ],
  [
    "...And Justice For All",
    "Luciano Pavarotti",
  ],
  [
    "...And Justice For All",
    "Leonard Bernstein & New York Philharmonic",
  ],
  [
    "...And Justice For All",
    "Boston Symphony Orchestra & Seiji Ozawa",
  ],
  [
    "...And Justice For All",
    "Aaron Copland & London Symphony Orchestra",
  ],
  [
    "...And Justice For All",
    "Ton Koopman",
  ],
  [
    "...And Justice For All",
    "Sergei Prokofiev & Yuri Temirkanov",
  ],
  [
    "...And Justice For All",
    "Chicago Symphony Orchestra & Fritz Reiner",
  ],
  [
    "...And Justice For All",
    "Orchestra of The Age of Enlightenment",
  ],
  [
    "...And Justice For All",
    "Emanuel Ax, Eugene Ormandy & Philadelphia Orchestra",
  ],
  [
    "...And Justice For All",
    "James Levine",
  ],
  [
    "...And Justice For All",
    "Berliner Philharmoniker & Hans Rosbaud",
  ],
  [
    "...And Justice For All",
    "Maurizio Pollini",
  ],
  [
    "...And Justice For All",
    "Academy of St. Martin in the Fields, Sir Neville Marriner & William Bennett",
  ],
  [
    "...And Justice For All",
    "Gustav Mahler",
  ],
  [
    "...And Justice For All",
    "Felix Schmidt, London Symphony Orchestra & Rafael Frühbeck de Burgos",
  ],
  [
    "...And Justice For All",
    "Edo de Waart & San Francisco Symphony",
  ],
  [
    "...And Justice For All",
    "Antal Doráti & London Symphony Orchestra",
  ],
  [
    "...And Justice For All",
    "Choir Of Westminster Abbey & Simon Preston",
  ],
  [
    "...And Justice For All",
    "Michael Tilson Thomas & San Francisco Symphony",
  ],
  [
    "...And Justice For All",
    "Chor der Wiener Staatsoper, Herbert Von Karajan & Wiener Philharmoniker",
  ],
  [
    "...And Justice For All",
    "The King's Singers",
  ],
  [
    "...And Justice For All",
    "Berliner Philharmoniker & Herbert Von Karajan",
  ],
  [
    "...And Justice For All",
    "Sir Georg Solti, Sumi Jo & Wiener Philharmoniker",
  ],
  [
    "...And Justice For All",
    "Christopher O'Riley",
  ],
  [
    "...And Justice For All",
    "Fretwork",
  ],
  [
    "...And Justice For All",
    "Amy Winehouse",
  ],
  [
    "...And Justice For All",
    "Calexico",
  ],
  [
    "...And Justice For All",
    "Otto Klemperer & Philharmonia Orchestra",
  ],
  [
    "...And Justice For All",
    "Yehudi Menuhin",
  ],
  [
    "...And Justice For All",
    "Philharmonia Orchestra & Sir Neville Marriner",
  ],
  [
    "...And Justice For All",
    "Academy of St. Martin in the Fields, Sir Neville Marriner & Thurston Dart",
  ],
  [
    "...And Justice For All",
    "Les Arts Florissants & William Christie",
  ],
  [
    "...And Justice For All",
    "The 12 Cellists of The Berlin Philharmonic",
  ],
  [
    "...And Justice For All",
    "Adrian Leaper & Doreen de Feis",
  ],
  [
    "...And Justice For All",
    "Roger Norrington, London Classical Players",
  ],
  [
    "...And Justice For All",
    "Charles Dutoit & L'Orchestre Symphonique de Montréal",
  ],
  [
    "...And Justice For All",
    "Equale Brass Ensemble, John Eliot Gardiner & Munich Monteverdi Orchestra and Choir",
  ],
  [
    "...And Justice For All",
    "Kent Nagano and Orchestre de l'Opéra de Lyon",
  ],
  [
    "...And Justice For All",
    "Julian Bream",
  ],
  [
    "...And Justice For All",
    "Martin Roscoe",
  ],
  [
    "...And Justice For All",
    "Göteborgs Symfoniker & Neeme Järvi",
  ],
  [
    "...And Justice For All",
    "Itzhak Perlman",
  ],
  [
    "...And Justice For All",
    "Michele Campanella",
  ],
  [
    "...And Justice For All",
    "Gerald Moore",
  ],
  [
    "...And Justice For All",
    "Mela Tenenbaum, Pro Musica Prague & Richard Kapp",
  ],
  [
    "...And Justice For All",
    "Emerson String Quartet",
  ],
  [
    "...And Justice For All",
    "C. Monteverdi, Nigel Rogers - Chiaroscuro; London Baroque; London Cornett & Sackbu",
  ],
  [
    "...And Justice For All",
    "Nash Ensemble",
  ],
  [
    "...And Justice For All",
    "Philip Glass Ensemble",
  ],
]